dotnet/samples/TeamsChefBot/TeamsChefBot.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+	<ProjectCapability Include="TeamsFx" />
+  </ItemGroup>
+	
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.20.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.20.0" />
+    <PackageReference Include="Microsoft.Bot.Connector" Version="4.20.0" />
+    <PackageReference Include="Microsoft.TeamsAI" Version="1.0.0" />
+  </ItemGroup>
+
+<ItemGroup>
+  <Content Include="Prompts\Chat\skprompt.txt">
+    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+  </Content>
+</ItemGroup>
+
```

Comment:
Just notice that TeamsFx folders (appPackage, env, infra, ...) should be exclude from output. E.g., adding this?
``` xml
  <!-- Exclude Teams Toolkit files from build output, but can still be viewed from Solution Explorer -->
  <ItemGroup>
    <Content Remove="appPackage/**/*" />
    <None Include="appPackage/**/*" />
    <None Include="env/**/*" />
    <Content Remove="infra/**/*" />
    <None Include="infra/**/*" />
  </ItemGroup>
```

Reply:
Good point. Updated. Seems our TTK template should do the same thing.